packages/modules/desktop-viewer-react/package.json

```diff
@@ -23,7 +23,7 @@
     "rebuild": "npm run clean && npm run build"
   },
   "dependencies": {
-    "@itwin/viewer-react": "3.0.0-dev.9"
+    "@itwin/viewer-react": "workspace:*"
```

Comment:
curious, how does this work with rush publish? Is rush smart enough to know to replace it in the published package but not write back to the repo?

Reply:
Ok trust you there :-)

Reply:
ill test it out locally with a dry run before merging into master

Reply:
tested with a next release and works as expected